### ISSUE TITLE
Currently supported H265 video format?

### ISSUE THREAD

**Issue by MuZiLee** [Bilibili/ijkplayer#1261]
Currently supported H265 video format?
eg:32k*32k,64k*64,128k*128 (optional)

**Comment by bbcallen** [comment]
Yes, but you need to modify config/module-lite.sh to enable it yourself.

**Comment by MuZiLee** [comment]
thank!
But I don't set to config/module-lite.sh the explanation, what can I do to help my information?

**Comment by MuZiLee** [comment]
What I mean is that the setting parameters of config/module-lite.sh is too much, only understand a part. Do you have any more comments convenient set is introduced?

**Comment by bbcallen** [comment]
run './configure --help' in FFmpeg's directory.

**Comment by MuZiLee** [comment]
@bbcallen 
Excuse me,In the module - lite - hevc. ShExport 
COMMON_FF_CFG_FLAGS = "$COMMON_FF_CFG_FLAGS - enable - parser = havoc" 
and 
COMMON_FF_CFG_FLAGS = "$COMMON_FF_CFG_FLAGS - enable - decoder = libx265" (my add)

What is the difference between?Both are equivalent?

**Comment by bbcallen** [comment]
Wrong, you need "--enable-decoder=hevc", not "--enable-decoder=libx265"
